### PR TITLE
py-pygraphviz: Add version 1.7

### PR DESCRIPTION
**Pull request by haralmha**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pygraphviz/package.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from spack import *
+
+
+class PyPygraphviz(PythonPackage):
+    """Python interface to Graphviz"""
+
+    homepage = "https://pygraphviz.github.io/"
+    pypi     = "pygraphviz/pygraphviz-1.7.zip"
+
+    maintainer = 'haralmha'
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pygraphviz/package.py]:
```suggestion
    maintainers = ['haralmha']
```

File: var/spack/repos/builtin/packages/py-pygraphviz/package.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    version('1.7', sha256='a7bec6609f37cf1e64898c59f075afd659106cf9356c5f387cecaa2e0cdb2304')
+
+    depends_on('python@3.7:3.9')
+    depends_on('py-setuptools')
+    depends_on('graphviz')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pygraphviz/package.py]:
```suggestion
    depends_on('python@3.7:', type=('build', 'run'))
    depends_on('py-setuptools', type='build')
    depends_on('graphviz')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — DISMISSED** [review]

**Comment by adamjstewart** [comment]
@spackbot re-run pipeline

**Comment by iarspider** [comment]
@spackbot re-run pipeline

**Comment by adamjstewart** [comment]
@spackbot re-run pipeline

**Comment by iarspider** [comment]
@spackbot re-run pipeline

**Comment by adamjstewart** [comment]
@haralmha can you try rebasing on develop? I think GitLab is stuck with an old commit.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**